src/mugs.js

```diff
@@ -295,6 +295,61 @@ define([
 
             return nodeID;
         },
+        serialize: function () {
+            var mug = this,
+                data = {type: mug.__className};
+            _.each(mug.spec, function (spec, key) {
+                if (spec.presence === "notallowed") {
+                    return;
+                }
+                var value = mug.p[key];
+                if (spec.serialize) {
+                    value = spec.serialize(value, key, mug, data);
+                    if (!_.isUndefined(value)) {
+                        data[key] = value;
+                    }
+                } else if (!(
+                                value === null ||
+                                value === "" ||
+                                _.isUndefined(value) ||
```

Comment:
That does appear to work, so I changed it. See also comment on 39052ff929a5ce142fa4845b1ccf0bd5d20d8fcb. I'd prefer to simply use `!_.isEmpty(value)`, but unfortunately `_.isEmpty(true) === true`, which is a deal breaker.